frontend/src/app/organization/organization-contact/organization-contact.component.ts

```diff
@@ -16,6 +21,14 @@ export class OrganizationContactComponent implements OnChanges {
   submitDisabled = false;
   alertSuccessClosed = true;
   alertErrorClosed = true;
+  user$: Observable<User>;
+  getFullName;
+
+  constructor(private authService: AuthService, userService: UserService) {
```

Comment:
here instead putting these 2 lines in `constructor` You should be able to do:
```typescript
user$: Observable<User> = this.authService.user$;
getFullName = this.userService.getFullName;

constructor(private authService: AuthService, private userService: UserService) { }
```
(this `private` at the beginning of `userService` is crutial here)

That way You don't need to declare class attribute without types like:
```typescript
getFullName;
```

Reply:
done, resolve that.

Reply:
It's not yet resolved, You still have:
```
user$: Observable<User>;
getFullName;

constructor(private authService: AuthService, userService: UserService) {
  this.user$ = authService.user$;
  this.getFullName = userService.getFullName;
}
```
instead of:
```
user$: Observable<User> = this.authService.user$;
getFullName = this.userService.getFullName;

constructor(private authService: AuthService, private userService: UserService) { }
```

Reply:
ok I got it now.

Reply:
and for the user mode I deleted these 4 lines because it was duplicated. Now it looks like